src/connectionProvider.ts

```diff
@@ -55,7 +55,7 @@ export class ConnectionProvider implements azdata.ConnectionProvider {
 	}
 
 	changeDatabase(connectionUri: string, newDatabase: string): Thenable<boolean> {
-		throw new Error('Method not implemented.');
+		return true;
```

Comment:
This needs to return a Thenable - you should be returning `Promise.resolve(true)` (or you could make it async and change the return type to Promise<boolean> since a Promise is a Thenable)

Did you compile this locally? It should have errored out when doing that.